web/src/utils/filter-definitions.ts

```diff
@@ -448,6 +449,34 @@ export const getFilterDefinitions = (
         encoder: simpleFiltersEncoder('_HashId'),
         overlap: false
       },
+      {
+        id: 'pkt_drop_state',
+        name: t('Packet drop TCP state'),
+        category: FilterCategory.None,
+        component: FilterComponent.Autocomplete,
+        getOptions: cap10(getDropStateOptions),
+        validate: rejectEmptyValue,
+        hint: t('Specify a single TCP state.'),
+        examples: `${t('Specify a single TCP state name like:')}
+        - ${t('A _LINUX_TCP_STATES_H number like 1, 2, 3')}
+        - ${t('A _LINUX_TCP_STATES_H TCP name like ESTABLISHED, SYN_SENT, SYN_RECV')}`,
```

Comment:
Like you put a link to the drop reasons, we may have a link to these states definitions? https://github.com/torvalds/linux/blob/master/include/net/tcp_states.h